snowpack/src/commands/dev.ts

```diff
@@ -981,15 +987,15 @@ export async function startServer(
     },
     getUrlForFile: (fileLoc: string) => {
       const result = getUrlsForFile(fileLoc, config);
-      return result ? result[0] : result;
+      return result ? result[0] : null;
     },
     onFileChange: (callback) => (onFileChangeCallback = callback),
     getServerRuntime: (options) => getServerRuntime(sp, options),
     async shutdown() {
       watcher && (await watcher.close());
       server && server.close();
     },
-  } as SnowpackDevServer;
```

Comment:
This `as` was throwing everything off. Many things weren’t being typed correctly.